### PR TITLE
equivalences induced by concatenation

### PR DESCRIPTION
**Pull request by emilyriehl**
I need to use an equivalence between path spaces `y = z` and `x = y` induced by preconcatenation with a path `p : x = y`.

So this has been added to the end of the equivalences file.

I also replaced a proof of `equiv (x = y) (y = x)` with a better proof of the same fact that appeared later.

### CODE REVIEW

File: src/hott/01-paths.rzk.md
Comment (fredrik-bakke) [inline src/hott/01-paths.rzk.md:897]:
```suggestion

```

**Review by fredrik-bakke — APPROVED** [review]

**Comment by emilyriehl** [comment]
@fredrik-bakke done.

**Comment by emilyriehl** [comment]
Let me know if you approve ;)

**Review by fredrik-bakke — APPROVED** [review]
Looks good! 👍

**Comment by emilyriehl** [comment]
@fredrik-bakke could you "approve" rather than just comment. I can't merge until someone approves.

**Review by jonweinb — APPROVED** [review]
Approved

**Comment by fredrik-bakke** [comment]
> @fredrik-bakke could you "approve" rather than just comment. I can't merge until someone approves.

I did! Github just doesn't respect my opinion as much as Jonathan's 😔

**Pull request merged**
